script.js

```diff
@@ -1 +1,82 @@
-'use strict'
\ No newline at end of file
+'use strict'
+
+
+const bodySelect = document.querySelector('body')
+const secretN = document.querySelector('.secret')
+const userInput = document.querySelector('.userInput')
+const btnCheck = document.querySelector('.checkBtn')
+const btnReplay = document.querySelector('.replayBtn')
+const position = document.getElementById('position')
+const scoreSelect = document.getElementById('score')
+const highScoreSelect = document.getElementById('highScore')
+
+let secretNumber
+let score
+let highscore = 0;
+
+const setup = function (){
+    secretNumber = Math.trunc(Math.random() * 20) + 1
+    score = 20
+}
+
+setup()
```

Comment:
That's a great addition.

But maybe you could do better.

Can you not imagine an object holding score, highscore and responsible of
+ generating the secret
+ reset all this when desired ? 
+ update the high score 
+ and maybe more

I do, so you should.

Hint:
```js
const player {
"score": ...
"highscore": ...
"secret": ...
reset() { ...
}
```